src/main/kotlin/espressoLogcat/Enums.kt

```diff
@@ -1,15 +1,23 @@
 package espressoLogcat
 
-enum class Priority {
-    VERBOSE,
-    DEBUG,
-    INFO,
-    WARNING,
-    ERROR,
-    FATAL,
-    SILENT
+/**
+ * Priority, also known as level, to be used when querying logcat
+ * `abbr` is the first character of the string
+ */
+enum class Priority(val abbr: String) {
```

Comment:
nitpicky, but for readability sake it could be good to name the variable in a more descriptive way. like the point of the variable is that it maps to the associated logcat flag, not just that it's the abbreviation. You could also make that variable internal if you want so that you aren't exposing logcat stuff publically

Reply:
Both are done in 750711e